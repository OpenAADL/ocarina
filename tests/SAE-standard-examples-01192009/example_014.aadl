-- test 6.5 Virtual Buses draft-01192009

package Hardware
-- add for test
public
-- end add

bus Ethernet
end Ethernet;
virtual bus IP_TCP
end IP_TCP;
virtual bus HTTP
properties
  Allowed_Bus_Binding_Class => virtual bus IP_TCP;
end HTTP;
processor PowerPC
end PowerPC;
processor implementation PowerPC.Linux
subcomponents
IP_TCP: virtual bus IP_TCP;
end PowerPC.Linux;
end Hardware;