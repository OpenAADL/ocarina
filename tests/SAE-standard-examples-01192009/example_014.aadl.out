example_014.aadl:14:32: parsing Property_Expression, not allowed in aadl v2
example_014.aadl:15:01: parsing Property_Association or Contained_Property_Association, token ';' is expected, found keyword 'end'
example_014.aadl:13:01: parsing Properties, list is empty
example_014.aadl:15:09: parsing Package_Specification, Empty packages are not allowed
example_014.aadl:16:01: parsing AADL_Declaration, not allowed in aadl v2
example_014.aadl:18:01: parsing AADL_Declaration, not allowed in aadl v2
example_014.aadl:21:01: parsing AADL_Declaration, unexpected keyword 'end'
example_014.aadl:22:01: parsing AADL_Declaration, unexpected keyword 'end'
Cannot parse AADL specifications
